stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkPeerManager.java

```diff
@@ -50,9 +53,20 @@ public AsyncPrettyPrinterRegistry getAsyncPrettyPrinterRegistry() {
     return mAsyncPrettyPrinterRegistry;
   }
 
+  public void setPrettyPrinterInitializer(@NonNull AsyncPrettyPrinterInitializer initializer) {
+    if (initializer == null) {
```

Comment:
you can just use `Util.throwIfNull(initializer)`. Don't really need the `@NonNull` annotation.